packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -62,6 +65,18 @@ export interface ParticipantItemStrings {
   mutedIconLabel: string;
   /** placeholder text for participants who does not have a display name*/
   displayNamePlaceholder?: string;
+  /* @conditional-compile-remove(one-to-n-calling) */
+  /* @conditional-compile-remove(PSTN-calls) */
+  /** String shown when `participantState` is `Connecting` */
+  participantStateConnecting?: string;
+  /* @conditional-compile-remove(one-to-n-calling) */
+  /* @conditional-compile-remove(PSTN-calls) */
```

Comment:
how do double conditional compiled work? Is this an AND or an OR?
My assumption is the implementation will be an AND (both of these features must be stabilized for this feature to be public), but we would want an OR (if either of these features are stabilized we want this feature public)

Reply:
So if either of these feature is stabilized, we remove both the flags

Because the assumption is that if this is stable for one feature, it is stable for another as well

Reply:
We chatted offline, we don't have support for this so whoever stabilizes this feature first needs to ensure they delete both

Reply:
Filed a bug for tracking this [Bug 2946816](https://skype.visualstudio.com/SPOOL/_workitems/edit/2946816): [Web] We do not support OR conditional compilation flags